RELEASING.adoc

```diff
@@ -1,35 +1,56 @@
 = Spring Cloud GCP release process
 
-The actual release process is triggered by the release admin and happens together with the rest of the Spring Cloud release.
-In order to simplify that process, here is a list of prerequisites to be performed before asking the release admin to run the release scripts.
+The actual release process is triggered by the release admin.
+As opposed to the 1.x releases, the 2.0 release happens independent of the Spring Cloud release train.
+To simplify the process, perform this list of prerequisites before asking the release admin to run the release scripts.
 
-. Ensure that modules that shouldn't be released have the `maven-deploy-plugin` configuration set to `<skip>true</skip>`.
+. Ensure that modules that shouldn't be released are not included in the `<modules>` block in the `release` profile of `pom.xml`.
 For GA releases this should be configured under the `central` Maven profile.
 +
 [source,xml]
 ----
-<build>
-    <plugins>
-        <plugin>
-            <artifactId>maven-deploy-plugin</artifactId>
-            <configuration>
-                <skip>true</skip>
-            </configuration>
-        </plugin>
-    </plugins>
-</build>
+<profile>
+    <id>release</id>
+    <modules>
+        <!-- POM Modules-->
+        <module>spring-cloud-gcp-starters</module>
+        <module>spring-cloud-gcp-dependencies</module>
+
+        <!-- Code Modules-->
+        <module>spring-cloud-gcp-autoconfigure</module>
+        <module>spring-cloud-gcp-bigquery</module>
+        <module>spring-cloud-gcp-cloudfoundry</module>
+        <module>spring-cloud-gcp-core</module>
+        <module>spring-cloud-gcp-data-datastore</module>
+        <module>spring-cloud-gcp-data-firestore</module>
+        <module>spring-cloud-gcp-data-spanner</module>
+        <module>spring-cloud-gcp-logging</module>
+        <module>spring-cloud-gcp-pubsub</module>
+        <module>spring-cloud-gcp-pubsub-stream-binder</module>
+        <module>spring-cloud-gcp-security-iap</module>
+        <module>spring-cloud-gcp-storage</module>
+        <module>spring-cloud-gcp-secretmanager</module>
+        <module>spring-cloud-gcp-security-firebase</module>
+        <module>spring-cloud-gcp-vision</module>
```

Comment:
Maybe a `...` under Code modules, too, so we don't have to keep it perfectly synchronized.

Reply:
Done.